src/pkg/retention/launcher.go

```diff
@@ -15,6 +15,7 @@
 package retention
 
 import (
+	"context"
 	"fmt"
 	beegoorm "github.com/astaxie/beego/orm"
 	"github.com/goharbor/harbor/src/lib/orm"
```

Comment:
These two imports can be avoided:
`beegoorm "github.com/astaxie/beego/orm"`
`"github.com/goharbor/harbor/src/lib/orm"`